### ISSUE TITLE
Office level enumeration

### ISSUE THREAD

**Issue by indymag** [votinginfoproject/vip-specification#290]
In VIP 5.0 the Office element doesn't contain an indicator about the "level" of government the office operates in.  This would be useful for applications that wish to filter or sort contests in a predictable way.  Example levels that would be useful in the US are:
* FEDERAL
* STATEWIDE
* CITYWIDE

An internationalized enumeration of levels can be seen in the Civic Info API documentation on levels:
https://developers.google.com/civic-information/docs/v2/elections/voterInfoQuery

**Comment by jdmgoogle** [comment]
@pstenbjorn How easy/difficult would it be to get jurisdictions to use this and apply it consistently and correctly (where "this" is the set of enumerations in Chetan's link)?

@johnpwack What are your thoughts on including this enumeration in version 1.1 of NIST's ERR spec?

**Comment by MariaDemocracyWorks** [comment]
Is this significantly different from the enumerations for Electoral District, which are fairly difficult to get jurisdictions to consistently apply?

**Comment by jdmgoogle** [comment]
No, it's a different set of enumerations which will be fairly difficult to get jurisdictions to consistently apply. :)

**Comment by jungshadow** [comment]
After @jdmgoogle's comment, I can only imagine @MariaDemocracyWorks and the rest of the DW office are like ![Animation of Arrested Development characters looking dramatically at each other](http://bukk.it/drama.gif)

**Comment by jdmgoogle** [comment]
Given that something like this will not be making it into the current version of the NIST standard, I'm punting this back to 5.2.

**Comment by jswiesner** [comment]
If we move forward with https://github.com/votinginfoproject/vip-specification/issues/350, we'd have the Office's jurisdiction represented by an ElectoralDistrict element. That would make the office's level synonymous with the [DistrictType](https://vip-specification.readthedocs.io/en/latest/built_rst/xml/enumerations/district_type.html#multi-xml-district-type) from Office.JurisdictionId.type. Thoughts on closing this issue in favor of Office.JurisdictionId.type?

**Comment by jdmgoogle** [comment]
Thoughts, @afsmythe et al?
